src/collections_16March2023/HashSetFeature.java

```diff
@@ -0,0 +1,25 @@
+package collections_16March2023;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class HashSetFeature {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Set<String>hashset = new HashSet<>();
```

Comment:
remove the String datatype from set and try to print

Reply:
Done